lib/features/thread/presentation/mixin/base_email_item_tile.dart

```diff
@@ -55,6 +55,43 @@ mixin BaseEmailItemTile {
     }
   }
 
+  Widget buildIconAvatarSwipe(
```

Comment:
please do it as a `stateless widget`

Reply:
should is make it inside new file as a stateless widget ?